server/feeds/sql/user-feed-activity.sql

```diff
@@ -9,21 +9,24 @@ SELECT
     TO_CHAR(last_update_timestamp, 'YYYY-MM-DD"T"HH24:MI:SS.00"Z"') as thread_last_activity_at,
     thread_details.default_view,
     -- Amount details
-    COALESCE(posts_amount, 0) as thread_total_posts_amount,
-    COALESCE(threads_amount, 0) as thread_direct_threads_amount,
-    COALESCE(comments_amount, 0) as thread_total_comments_amount,
-    COALESCE(new_posts_board_amount, 0) as thread_new_posts_amount,
-    COALESCE(new_comments_board_amount, 0) as thread_new_comments_amount,
+    COALESCE(posts_amount, 0)::integer as thread_total_posts_amount,
+    COALESCE(threads_amount, 0)::integer as thread_direct_threads_amount,
+    COALESCE(comments_amount, 0)::integer as thread_total_comments_amount,
+    COALESCE(new_posts_board_amount, 0)::integer as thread_new_posts_amount,
+    COALESCE(new_comments_board_amount, 0)::integer as thread_new_comments_amount,
     COALESCE(muted, FALSE) as muted,
     COALESCE(hidden, FALSE) as hidden,
+    FALSE as starred,
     -- Post details (DbPostType)
     first_post_string_id as post_id,
     thread_external_id as parent_thread_id,
     NULL as parent_post_id,
     board_slug as parent_board_slug,
     board_external_id as parent_board_id,
+    'everyone' as anonymity_type,
+    'text' as type,
```

Comment:
im assuming that it fails without these? lmao

Reply:
It does lmao